src/styles/blocks/_pages.scss

```diff
@@ -0,0 +1,103 @@
+.page {
+  font-family: Avenir, sans-serif;
+  scroll-behavior: smooth;
+  @include font(16px, 500, 24px, $color-black);
+
+  &__body {
+    margin: 0;
+    background-color: $color-gray4;
+
+    &--with-menu {
+      overflow: hidden;
+    }
+  }
+
+  &__menu {
+    position: fixed;
+    top: 0;
+    left: 0;
+    right: 0;
+    z-index: 1;
+    // overflow: hidden;
+
+    transform: translateX(-100%);
+    transition: transform 1s ease;
```

Comment:
Pay attention to comments, make a variable for transition-duration and use it everywhere